spec/unit/matrix-client.spec.ts

```diff
@@ -52,6 +58,29 @@ jest.mock("../../src/webrtc/call", () => ({
     supportsMatrixCall: jest.fn(() => false),
 }));
 
+enum AnsiColorCode {
+    Red = 31,
+    Green = 32,
+    Yellow = 33,
+}
+// Add color to text in the terminal output
+function decorateStringWithAnsiColor(inputString: string, decorationColor: AnsiColorCode): string {
+    return `\x1b[${decorationColor}m${inputString}\x1b[0m`;
+}
```

Comment:
I'm hesitant to add console colouring code as there's about a million different runtimes we have to consider - this would be best placed in a dedicated PR to consider impact, avoiding a review delay here.